lib/conversions.js

```diff
@@ -1,3 +1,5 @@
+'use strict';
```

Comment:
not needed, we lint with standardjs instead

Reply:
yeah

Reply:
ok